src/controllers/update/overseas.entity.review.controller.ts

```diff
@@ -31,7 +31,8 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
 export const post = (req: Request, res: Response, next: NextFunction) => {
   try {
     logger.debugRequest(req, `POST ${config.OVERSEAS_ENTITY_REVIEW_PAGE}`);
-    return res.redirect(config.UPDATE_CHECK_YOUR_ANSWERS_PAGE);
+    // TO-DO: Place of MO type page not agreed upon, once agreed this will change
+    return res.redirect(config.UPDATE_MANAGING_OFFICER_TYPE_PAGE);
```

Comment:
This will need to go to BOs first. Can sort out these links just before merging.